### ISSUE TITLE
Load MapReduce result partitions in an S3 bucket

### ISSUE THREAD

**Issue by tuanminhnguyen** [cloudyr/aws.s3#167]
Hello,

Would it be possible to load into R an S3 bucket which contains partitions of the result of, e.g., a Hive query? Currently it looks like the only way to do it is to call get_object for each partition, create a rawToChar connection, read.csv using that connection, and merge the individual df's. Am I missing anything? 

Thanks for your help!

- T

**Comment by leeper** [comment]
That's correct. If it would be helpful to have a higher-level function for that procedure, please send a PR or let me know exactly what your use case looks like so I can try to implement something.

**Comment by tuanminhnguyen** [comment]
Thank you, leeper! I would like to load into R the partition files of the results of a MapReduce query. These files are in tabular format, stored in one S3 bucket as text files, with delimiter being comma, tab, or Ctrl-A. Once loaded into R, these files are merged into once single object (data.table format would be preferable). That is my use case. If you could implement something for that, that would be great!
